src/plots/polar/polar.js

```diff
@@ -937,6 +940,8 @@ proto.updateRadialDrag = function(fullLayout) {
     var tx = cx + (radius + bl2) * Math.cos(angle0);
     var ty = cy - (radius + bl2) * Math.sin(angle0);
 
+    // TODO add 'inner' drag box when innerRadius > 0 !!
```

Comment:
not quite sure what this means, but if it's something like the clamped inner edge still jumps to be a circle at the center then yeah, we'd better avoid that.

Reply:
Oh right! Yes, we should add that. Would be confusing if there's room for one but it's not there.

Reply:
![peek 2018-09-07 16-41](https://user-images.githubusercontent.com/6675409/45242135-e9265780-b2bc-11e8-9ae3-6d4a6dc03fca.gif)

I'm really liking this effect.

Reply:
Added in https://github.com/plotly/plotly.js/pull/2977/commits/d46614605b5dd410c255df625460be9d8da15ad7